hr_expense_payment_difference/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module extends the functionality of hr expense to allow you to create
```

Comment:
This module extend the functionality of payment wizard in hr expense, to also allow you to enter the payment difference amount.